yt_dlp/extractor/gamejolt.py

```diff
@@ -0,0 +1,540 @@
+# coding: utf-8
+import itertools
+import json
+import math
+
+from .common import InfoExtractor
+from ..compat import compat_urllib_parse_unquote
+from ..utils import (
+    determine_ext,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    try_get
+)
+
+
+class GameJoltBaseIE(InfoExtractor):
+    _API_BASE = 'https://gamejolt.com/site-api/'
+
+    def _call_api(self, endpoint, *args, **kwargs):
+        kwargs.setdefault('headers', {}).update({'Accept': 'image/webp,*/*'})
+        return self._download_json(self._API_BASE + endpoint, *args, **kwargs)['payload']
+
+    def _parse_content_as_text(self, content):
+        outer_contents, joined_contents = content.get('content') or [], []
+        for outer_content in outer_contents:
+            if outer_content.get('type') != 'paragraph':
+                joined_contents.append(self._parse_content_as_text(outer_content))
+                continue
+            inner_contents, inner_content_text = outer_content.get('content') or [], ''
+            for inner_content in inner_contents:
+                if inner_content.get('text'):
+                    inner_content_text += inner_content['text']
+                elif inner_content.get('type') == 'hardBreak':
+                    inner_content_text += '\n'
+            joined_contents.append(inner_content_text)
+
+        return '\n'.join(joined_contents)
+
+    def _get_comments(self, post_num_id, post_hash_id):
+        sort_by, scroll_id = self._configuration_arg('comment_sort', ['hot'])[0], -1
```

Comment:
add it to readme

Reply:
Now that I think about it, the user would need to be set the arg for every subclassed extractor right now (since it's calling `self`)